backend/src/main/java/ch/puzzle/okr/controller/UnitController.java

```diff
@@ -0,0 +1,80 @@
+package ch.puzzle.okr.controller;
+
+import ch.puzzle.okr.dto.UnitDto;
+import ch.puzzle.okr.mapper.UnitMapper;
+import ch.puzzle.okr.models.Unit;
+import ch.puzzle.okr.service.authorization.UnitAuthorizationService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import java.util.List;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("api/v2/units")
+public class UnitController {
+    private final UnitAuthorizationService unitAuthorizationService;
+    private final UnitMapper unitMapper;
+
+    public UnitController(UnitAuthorizationService unitAuthorizationService, UnitMapper unitMapper) {
+        this.unitAuthorizationService = unitAuthorizationService;
+        this.unitMapper = unitMapper;
+    }
+
+    @Operation(summary = "Get Units by User", description = "Retrieves a list of units associated with the currently authenticated user.")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "Successfully retrieved user's units", content = {
+                    @Content(mediaType = "application/json", schema = @Schema(implementation = UnitDto.class)) }),
+            @ApiResponse(responseCode = "400", description = "Bad request", content = @Content) })
```

Comment:
```suggestion
 })
```
Remove as 400as those are not relevant for GET

Reply:
fixed